crates/proto/src/version/v662/packets/add_actor.rs

```diff
@@ -0,0 +1,38 @@
+use bedrockrs_macros::{gamepacket, ProtoCodec};
+use crate::version::v662::types::{ActorLink, ActorRuntimeID, ActorUniqueID, DataItem, PropertySyncData, Vec2, Vec3};
+
+#[derive(ProtoCodec)]
+struct AttributeEntry {
+    pub attribute_name: String,
+    #[endianness(le)]
+    pub min_value: f32,
+    #[endianness(le)]
+    pub current_value: f32,
+    #[endianness(le)]
+    pub max_value: f32,
+}
+
+#[gamepacket(id = 13)]
+#[derive(ProtoCodec)]
+pub struct AddActorPacket {
+    pub target_actor_id: ActorUniqueID,
+    pub target_runtime_id: ActorRuntimeID,
+    pub actor_type: String,
+    pub position: Vec3,
```

Comment:
These `Vec3` and `Vec2` types need an extra template arg, this is usually just `Vec3<f32>` and `Vec2<f32>`. You also need to specify the endianess.. which is normally little (`le`)